projecty/test/testCases/FetchTweetsTest.java

```diff
@@ -8,37 +8,57 @@
 import org.junit.runner.RunWith;
 import org.mockito.junit.MockitoJUnitRunner;
 import twitter4j.*;
-import twitter4j.conf.ConfigurationBuilder;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Date;
 import java.util.List;
 
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
+
 @RunWith(MockitoJUnitRunner.class)
 public class FetchTweetsTest {
-    ConfigurationBuilder configurationBuilder;
+
     TwitterFactory twitterFactory;
     FetchTweets fetchTweets;
     Twitter twitter;
     Service service;
+    TwitterData twitterData;
     ResponseList responseList;
     Status s1;
     Status s2;
     Status s3;
+    SimpleDateFormat dateFormat = new SimpleDateFormat("dd-MM-yyyy HH:mm:ss");
+    String twitterHandle="@Kgaur624";
+    String name="Kartik";
+    String message="try";
+    String profileImageUrl="www.KartikGour.com";
+    Date created;
+    String date;
+    {
+        try {
+            created = dateFormat.parse("16-11-2021 01:03:00");
+        } catch (ParseException E) {
+            E.printStackTrace();
+        }
+        date = dateFormat.format(created);
+    }
 
     @Before
     public void setUp() {
         twitter= mock(Twitter.class);
-        //service= mock(Service.class);
```

Comment:
remove the unwanted lines

Reply:
hello @vglalitha
as you told i have done these changes